oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -130,44 +188,29 @@ For example, in the `edx-platform`_ repo, the file might look like:
 Rationale
 =========
 
+Keys Rationale
+--------------
+
 The keys in ``openedx.yaml`` were derived from existing repository metadata collected by edx.org.
 
+OEPs Rationale
+--------------
+
 The design of the ``oeps`` dictionary was guided by a couple of use cases:
 
 1. Adding a new OEP that few repositories will support, initially. We shouldn't need to update all repositories to detect if they are in accordance, we should be able to assume that they aren't, or automatically detect whether they are.
 2. Repositories may have specific requirements that force them to not implement a best practice. The tools should be able to present that reasoning to anyone looking across repositories, and the reasons should be documented in the repositories themselves.
 3. As much as possible, Best Practices should be autodetected, but because they will often involve a judgement call, autodetection shouldn't be mandatory.
 
+Ownership Rationale
+-------------------
 
-Change History
-==============
```

Comment:
Why are we deleting the change history?

Reply:
Our newer OEPs and OEP templates no longer require a `Change History` within the OEP.  The expectation is that folks can look at the OEP's Git commit history to catch up on its historical changes.

Having said that, if we believe this is a controversial change, I can revert this deletion and do so in its own PR.

Reply:
I think one of the over-arching bits of feedback we get from the community is, "it's hard to know what you've done."  OEPs change rarely. It won't be a burden to maintain a change history.

Reply:
@nedbat I'll go ahead and re-insert the change history in this PR.  We can determine outside of this OEP whether there is still value in keeping this history within the content of OEPs.